src/rules/noRequireImportsRule.ts

```diff
@@ -20,15 +20,29 @@ import * as ts from "typescript";
 
 import * as Lint from "../index";
 
+const OPTION_IGNORE_MODULE = "ignore-module";
+
 export class Rule extends Lint.Rules.AbstractRule {
     /* tslint:disable:object-literal-sort-keys */
     public static metadata: Lint.IRuleMetadata = {
         ruleName: "no-require-imports",
         description: "Disallows invocation of `require()`.",
         rationale: "Prefer the newer ES6-style imports over `require()`.",
         optionsDescription: "Not configurable.",
-        options: null,
-        optionExamples: [true],
+        options: {
+            items: {
+                properties: {
+                    "ignore-module": {
+                        type: "string",
+                    },
+                },
+                type: "object",
+            },
+            maxLength: 1,
+            minLength: 0,
+            type: "array",
+        },
+        optionExamples: [true, [true, { [OPTION_IGNORE_MODULE]: "(\\.html|\\.css)$" }]],
```

Comment:
I would actually prefer this to be a list of regex patterns. One big pattern like in `no-import-side-effect` is very hard to maintain.
#3504 also uses multiple patterns.